ipa-core/src/protocol/ipa_prf/prf_sharding/feature_label_dot_product.rs

```diff
@@ -94,19 +97,23 @@ impl InputsRequiredFromPrevRow {
         )
         .await?;
 
-        let capped_attributed_feature_vector = select(
-            ctx.narrow(&Step::ComputedCappedFeatureVector),
+        let condition: Replicated<Boolean, B> = Replicated::<Boolean, B>::expand(&capped_label);
```

Comment:
@andyleiserson - please help me. I am stuck on this ONE THING. I cannot figure out how to just take a Replicated<Boolean> and expand it into a Replicated<Boolean, B>... like just duplicate the bit B times...

Reply:
This doesn't work either. There is no `left` or `right` on `capped_label`, there is only `left_arr` and `right_arr`

Reply:
Sorry, `left` and `right` are methods on the `ReplicatedSecretSharing` trait, you need to import it.